docs/api.md

```diff
@@ -3,13 +3,13 @@
 ## `Botd.load`
```

Comment:
Can you also mention that botd has a server-side API?
create server_api.md and list available operations there.
For `poll` - we should say that it's possible to get the info on the server (via server-side API, which is preferred) and on the client, via `poll`. The responses should be identical and strongly typed.

Reply:
Fixed